### DISCUSSION TITLE
Just a review.

### DISCUSSION THREAD

**Discussion by rennroot** [jonsafari/mocp#35] [General]
I just want to say thank you. I only use your player. Great job.

**Comment by Spreadcat** [comment]
Came here for an issue after upgrading to Fedora 42 and getting "Segmentation fault". but.....

I never thanked you for all the years with the player. I'll try to compile it and see if that works further.

But for so long: Thank you as well for all the years with the MOC player. It worked great and did simply everything I asked for.
